ompi/mpi/c/iallreduce.c

```diff
@@ -93,6 +94,15 @@ int MPI_Iallreduce(const void *sendbuf, void *recvbuf, int count,
         OMPI_ERRHANDLER_CHECK(err, comm, err, FUNC_NAME);
     }
 
+    /* MPI standard says that reductions have to have a count of at least 1,
+     * but some benchmarks (e.g., IMB) calls this function with a count of 0.
```

Comment:
No, I `ompi-tests` has the 3.2.4. The latest is 4.1, which contains RMA and NBC benchmarks - might be good to add to that repo, and make an MTT section for it.
 * https://software.intel.com/en-us/articles/intel-mpi-benchmarks